agent/engine/task_manager.go

```diff
@@ -489,8 +494,27 @@ func (mtask *managedTask) cleanupTask(taskStoppedDuration time.Duration) {
 		cleanupTimeBool <- true
 		close(cleanupTimeBool)
 	}()
+	// wait for the cleanup time to elapse, signalled by cleanupTimeBool
 	for !mtask.waitEvent(cleanupTimeBool) {
 	}
+	stoppedSentBool := make(chan bool)
+	go func() {
```

Comment:
I meant "backend service" keep on starting a new task, and these task get stuck in "cleanup" state for 72 hours...,  eventually will agent run out of memory?